pkgs/development/compilers/elm/default.nix

```diff
@@ -111,17 +153,51 @@ let
             mkdir -p unpacked_bin
             ln -sf ${elm-instrument}/bin/elm-instrument unpacked_bin/elm-instrument
           '';
+          meta = with stdenv.lib; {
+            description = "Work in progress - Code coverage tooling for Elm";
+            homepage = "https://github.com/zwilias/elm-coverage";
+            license = licenses.bsd3;
+            maintainers = [ maintainers.turbomack ];
+          };
         });
 
       create-elm-app = patchNpmElm (patchBinwrap [elmi-to-json]
-        nodePkgs.create-elm-app);
+        nodePkgs.create-elm-app) // {
+          meta = with stdenv.lib; {
+            description = "Create Elm apps with no build configuration.";
```

Comment:
```suggestion
            description = "Create Elm apps with no build configuration";
```

Reply:
thanks I missed this one. One thing I did to the original taglines taken from upstream was to remove `.` if it was single sentence :+1: